docs/quickstarts/packet.md

```diff
@@ -5,262 +5,243 @@
-mycluster-worker-pool-1-worker-2              True     KubeletReady    kubelet is posting ready status
+Node                             Ready    Reason          Message                            
+                                                                                             
+lokomotive-demo-controller-0       True     KubeletReady    kubelet is posting ready status    
+lokomotive-demo-pool-1-worker-0    True     KubeletReady    kubelet is posting ready status    
+lokomotive-demo-pool-1-worker-1    True     KubeletReady    kubelet is posting ready status    
 
 Success - cluster is healthy and nodes are ready!
 ```
 
-Use the generated `kubeconfig` file to access the Kubernetes cluster and list nodes.
+## Verification
+
+Use the generated `kubeconfig` file to access the cluster:
 
 ```console
-export KUBECONFIG=./lokomotive-assets/cluster-assets/auth/kubeconfig
+export KUBECONFIG=$(pwd)/assets/cluster-assets/auth/kubeconfig
 kubectl get nodes
 ```
 
+Sample output:
+
+```console
+NAME                            STATUS   ROLES    AGE   VERSION
+lokomotive-demo-controller-0      Ready    <none>   33m   v1.17.4
+lokomotive-demo-pool-1-worker-0   Ready    <none>   33m   v1.17.4
+lokomotive-demo-pool-1-worker-1   Ready    <none>   33m   v1.17.4
+```
+
 ## Using the cluster
 
-At this point you have access to the Kubernetes cluster and can use it!
-If you don't have Kubernetes experience you can check out the [Kubernetes
-Basics official
-documentation](https://kubernetes.io/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/)
-to learn about its usage.
+At this point you should have access to a Lokomotive cluster and can use it to deploy applications.
 
-**Note**: Lokomotive sets up a pretty restrictive Pod Security Policy that
-disallows running containers as root by default, check the [Pod Security Policy
-documentation](../concepts/securing-lokomotive-cluster.md#cluster-wide-pod-security-policy)
-for more details.
+If you don't have any Kubernetes experience, you can check out the [Kubernetes
+Basics](https://kubernetes.io/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/) tutorial.
+
+>NOTE: Lokomotive uses a relatively restrictive Pod Security Policy by default. This policy
+>disallows running containers as root. Refer to the
+>[Pod Security Policy documentation](../concepts/securing-lokomotive-cluster.md#cluster-wide-pod-security-policy)
+>for more details.
 
 ## Cleanup
 
-To destroy the Lokomotive cluster, execute the following command:
+To destroy the cluster, execute the following command:
 
 ```console
 lokoctl cluster destroy --confirm
 ```
 
-You can safely delete the working directory created for this quickstart guide if you no longer
-require it.
+You can now safely delete the directory created for this guide if you no longer need it.
 
 ## Troubleshooting
 
-### Stuck at copy controller secrets
+In case the deployment fails, adding the `-v` flag to `lokoctl cluster apply` can provide useful
```

Comment:
We show path to the logs file if things are not run in verbose mode.

Reply:
Yes, I'm aware. I'm not fond of this UX - see https://github.com/kinvolk/lokomotive/issues/564.
In the current state I'm getting the feeling we want to run with `-v` from the start since I don't get the impression everything usually works smoothly on first attempt.

Anyway, sending the user to read a file from disk is :grimacing: IMO, and also the files are hard to read because they contain shell color info.

WDYT?

Reply:
We've opted for deploying with `-v` from the start.